server/middleware/populateCurrentUser.ts

```diff
@@ -8,6 +8,9 @@ export default function populateCurrentUser(userService: UserService): RequestHa
       if (res.locals.user) {
         const user = res.locals.user && (await userService.getUser(res.locals.user.token))
         if (user) {
+          if (!user.activeCaseLoadId) {
+            user.activeCaseLoadId = await userService.getActiveCaseLoadId(res.locals.user.token)
```

Comment:
`user.token` as you've already got `user` available.

Reply:
Hmm 🤔. Went through it with the debugger to check what's going on here. I think it _does_ have to be `res.locals.user.token` because the `user` here is the result of `userService.getUser()` which is different to the contents of `res.locals.user`. It's only the latter that has the `token`.